addon/components/denali-chip-group.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="chip-group {{attributes.class}}" ...attributes>
+  {{yield}}
```

Comment:
To strengthen the contract, can we also yield chip component?
```suggestion
  {{yield (hash 
    Chip=(component 'denali-chip')
  )}}
```

Reply:
As mentioned above to chris it seems odd to be yielding components that are already publicly exposed to the user elsewhere. It implies (to me) that there is a difference between the yielded component and the top level public one. 

And as for strengthening the API contract, it only strengthens the contract if the user is made aware of what is being yielded in docs, and would not have been made aware that `DenaliChip` is expected in `DenaliChipGroup`. In my opinion if only pushes the API documentation to a different place.

I'll yield it for now to conform to the rest of the project but It does feel weird to me.

In the case that later down the road there is a difference between these yielded components and the global ones then it will have been a good choice anyway.